### PR TITLE
fix(omr): preserve supported key metadata without false defaults

### PR DESCRIPTION
**Pull request by landfill**
## Purpose

First bounded deliverable from the Satie / Always With Me / Love Affair validation: fix traditional key metadata without changing recognized notes or playback. This does not resolve the wider recognition issues.

## Change

- Decode first key declaration for fifths -7..7, major/minor; absent mode retains the historical major-name convention.
- Omit unknown, malformed, unsupported or out-of-range key metadata instead of inventing C or aborting conversion.
- Keep the existing optional-string canonical contract; no schema object/version change or stored-score rewrite.
- Bound ASCII numeric text before integer conversion, including pathological digit lengths and valid long leading-zero forms.
- Add a raw converter CLI gate, full Python extraction matrix in CI, and canonical normalization tests.

## Evidence and review

- Regression-first: negative/invalid/minor tests failed before implementation. Independent Luna review exposed giant-number and Unicode-digit edge cases; both were reproduced and fixed.
- Sol implemented the initial contract; Luna independently reviewed it; coordinator reviewed/fixed boundaries and reran verification. Opus reviewed broader recognition priorities separately.
- Final 102 Jest suites / 972 tests PASS; 83 Python service tests PASS; independent typecheck/lint/build PASS (build itself skips type/lint).
- Retained actual XML replay: Always 647 notes unchanged with only key C→F; Satie 239/D and Love 411/E unchanged. Every non-key, non-generated field is equal.
- Decision D-050, phase OMR-Q1 and docs/recovery/validation/2026-09-06-key-metadata-implementation.md document contract and limits.

## Risks / exclusions / rollback

Unknown keys are now absent rather than always present as C; canonical/internal consumers already support absence, but raw external consumers should not assume the field exists. Only first declaration and traditional major/minor fifths are covered; no mode inference, key-change map, namespace expansion or transposing-part analysis.

No improvement to missing notes/bars, dots/ties, printed tempo recognition, repeats or hand assignment is claimed. Production remains unchanged. Rollback is reverting this converter-only PR or retaining the current production image; no data migration is needed.

## Summary by CodeRabbit

- **새 기능**
  - MusicXML의 조표 정보를 전통적인 이름으로 변환해 결과에 표시합니다.
  - 장조·단조 조표를 지원하며, 유효한 조표가 없으면 해당 필드를 생략합니다.

- **버그 수정**
  - 잘못되었거나 지원되지 않는 조표 정보가 있어도 음표 변환이 중단되지 않습니다.
  - 조표가 문서 순서에 따라 일관되게 처리됩니다.

- **문서**
  - 조표 메타데이터 보존 및 복구 절차를 문서화했습니다.

- **테스트**
  - 유효·누락·범위 초과·잘못된 조표 입력에 대한 검증을 추가했습니다.

### CODE REVIEW

File: docs/recovery/DECISIONS.md [hunk 1]
```diff
@@ -1898,3 +1898,32 @@
 - Tested: Same-PDF isolated automatic retry selected 9/8 in 28.604s; exported overflow bars 10→1. Runtime numeral module repeated the 3,048-label probe without non-9 proposals. Failure/budget/cancellation tests pass.
 - Not-tested: Full-score musical correctness, handwritten/unseen digit fonts and end-to-end genuine 6/8 PDF corpus. First-bar dot/tie defects and fresh baseline's late-positioned opening tempo remain unresolved; do not close #134 or mark its phase DONE for this partial repair.
 - Related: #134; validation/2026-09-06-recognition-reference-checkpoint.md
+
+## D-050: 읽을 수 없는 조표를 C로 꾸미지 않고 기존 optional 문자열 계약을 사용한다
+
+- Date: 2026-09-06
```
[the remark this answers is not in the document]
Comment (landfill) [reply docs/recovery/DECISIONS.md:1904]: The date is intentional and correct in the project/user timezone, Asia/Seoul (KST, UTC+09:00). Decision commit 5ee456663d0348e5f062ec5047f2624d6f0b5f8e has author/commit time 2026-09-06T04:10:25+09:00 (2026-09-05T19:10:25Z). The implementation commit is 2026-09-06T04:31:06+09:00. Thus 2026-09-06 is not a future project-local date; changing it to the UTC calendar date would make the recorded local date incorrect. Retaining the decision date; no code change is needed.
